index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
```

Comment:
- [ ] As the previous code reviewer asked, you should use the font-family from the template. I see that you imported it and I recommend you import other font-weights since the template uses more than one font-weight. If a linter issue appears you should google it or ask somebody to help, it should not prevent you from using the font.